cloud/amazon/iam_cert.py

```diff
@@ -228,6 +249,7 @@ def main():
         name=dict(default=None, required=False),
         cert=dict(default=None, required=False),
         key=dict(default=None, required=False),
+        key_contents=dict(default=None, required=False),
         cert_chain=dict(default=None, required=False),
         new_name=dict(default=None, required=False),
         path=dict(default='/', required=False),
```

Comment:
Instead of the failure occurring inside of  `def private_key_from_file_or_inline(module):`

the failure should be caught in the argument_spec as so

```python

argment_spec.update(dict(
    state=dict...
    #rest of the spec
),
mutually_exclusive=[['key','key_contents']]
)

```